src/assets/scripts/embed.njk

```diff
@@ -87,6 +98,21 @@ class WebringBanner extends HTMLElement {
             border-radius: 50%;
           }
 
+          .webring-banner_text {
+            line-height:0%;
+            -inkscape-font-specification:'Comfortaa, Normal';
+            font-variant-ligatures:normal;
+            font-variant-caps:normal;
+            font-variant-numeric:normal;
+            font-variant-east-asian:normal;
```

Comment:
Are those needed? I'm not up-to-date with web stuff, but they seem potentially superfluous.

Reply:
Most likely not, I just copied them over from the svg.